chart/flux/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 appVersion: "1.12.0"
-version: 0.9.0
+version: 0.10.0
```

Comment:
Please undo this change. The chart version will be bumped in a separate PR e.g. https://github.com/weaveworks/flux/pull/1924/files

Reply:
done